packages/nuxt/src/runtime/plugin.vue2.ts

```diff
@@ -10,6 +10,13 @@ export default (context: any, inject: any) => {
   context.app.pinia = pinia
   setActivePinia(pinia)
 
+  // add access to `$nuxt`
+  // @ts-expect-error: _p is internal
+  pinia._p.push(({ store }) => {
+    // make it non enumerable so it avoids any serialization and devtools
+    Object.defineProperty(store, '$nuxt', { value: context })
```

Comment:
Nuxt context (app) can be accessed with useNuxtApp() global. Any specific usecase for jt?

Reply:
People use to access `$fetch` and other properties. Should they not? I was quite requested back when I added it

Reply:
For Nuxt 2 yes made sense, but in Nuxt 3 (and with bridge), we have global `$fetch` and composables replacing the old `$nuxt` API.

Reply:
Should I mark it as deprecated then? Is there a page that makes sense to point in the jsdoc?

Reply:
It is (currently briefly) documented here: https://v3.nuxtjs.org/guide/going-further/nuxt-app

Since this PR is breaking change, I would drop it. `$nuxt` interface in Nuxt 2 is not same as `NuxtApp`. It might be confusing to keep (still your choice)

Reply:
(to be deprecated)